.github/workflows/build.yml

```diff
@@ -15,10 +16,10 @@ jobs:
     steps:
       - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4
 
-      - name: Use Node.js ${{ matrix.node-version }}
+      - name: Use Node.js 20.10.0
```

Comment:
I would leave this floating, any reason we are trying to bake this in?

Reply:
In one word? Laziness 😆 

I added the matrix now, see last commit. TY!